src/main.c

```diff
@@ -81,6 +81,7 @@ static void usage(int code, int keep_it_short) {
       "  -M               monochrome (don't colorize JSON);\n"
       "  -S               sort keys of objects on output;\n"
       "  --tab            use tabs for indentation;\n"
+      "  --strip-comments accept and strip C- or C++-style comments;\n"
```

Comment:
Maybe this should be `"  --strip-comments accept and strip comments from input;\n"`.  That way we could be format-neutral and add new JSON comment types later.